plot_data/templates.py

```diff
@@ -28,36 +28,61 @@
       background: #04AA6D;
       cursor: pointer;
     }
+
+    #buttons {
+      display: flex;
+      column-gap: 3rem;
+      row-gap: 0.5em;
+      flex-wrap: wrap;
+      margin-bottom: 0.5rem;
+      margin-top: 1rem;
+      margin: 0 auto;
+    }
+
+    #sub_button {
+      display: flex;
+      gap: 0.5rem;
+      flex-wrap: wrap;
+    }
+
     </style>
     <head>
         <script src=$core_path></script>
     </head>
     <div id="buttons">
-        <button name="mergeON" value="OK" type="button"
-        onclick="plot_data.switchMerge()"> Switch Point Merge </button> &nbsp;&nbsp;
-        <button name="Zoom" value="OK" type="button"
-        onclick="plot_data.switchZoom()"> Zoom Box </button>
-        <button name="Zoom+" value="OK" type="button" onclick="plot_data.zoomIn()"> Zoom+ </button>
-        <button name="Zoom-" value="OK" type="button" onclick="plot_data.zoomOut()"> Zoom- </button> &nbsp;&nbsp;
-        Cluster:&nbsp; <input type="range" class="slider" min="0" max="2500" value="1250"
-        onclick="plot_data.simpleCluster(value / 10000)"></input>
-        <button name="resetClusters" value="OK" type="button"
-        onclick="plot_data.resetClusters()"> Reset clusters </button> &nbsp;&nbsp;
-        <button name="resetView" value="OK" type="button"
-        onclick="plot_data.resetView()"> Reset view </button> &nbsp;&nbsp;
-        <button name="showPoints" value="OK" type="button"
-        onclick="plot_data.togglePoints()"> Show points </button>
-        <button name="switchOrientation" value="OK" type="button"
-        onclick="plot_data.switchOrientation()"> Change Disposition </button>
-        <button name="toogleAxes" value="OK" type="button"
-        onclick="plot_data.htmlToggleAxes()"> Show / Hide Axes </button>
-        <button name="logScale" value="OK" type="button"
-        onclick="plot_data.switchLogScale()"> Log Scale</button>
+        <div id="sub_button">
+            <button name="mergeON" value="OK" type="button"
+            onclick="plot_data.switchMerge()"> Switch Point Merge </button>
+        </div>
+        <div id="sub_button">
+            <button name="Zoom" value="OK" type="button"
+            onclick="plot_data.switchZoom()"> Zoom Box </button>
+            <button name="Zoom+" value="OK" type="button" onclick="plot_data.zoomIn()"> Zoom+ </button>
+            <button name="Zoom-" value="OK" type="button" onclick="plot_data.zoomOut()"> Zoom- </button>
+        </div>
+        <div id="sub_button">
+            Cluster: <input type="range" class="slider" min="0" max="2500" value="1250"
+            onclick="plot_data.simpleCluster(value / 10000)">
```

Comment:
/ missing at the end of the input selector

<input ...rest />

Reply:
Done